cmds/vpdbootmanager/addentry/add.go

```diff
@@ -0,0 +1,45 @@
+package addentry
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/vpd"
+)
+
+const usage = `Usage:
+boot add [netboot [dhcpv6|dhcpv4] [MAC] | localboot [grub|path [Device GUID] [Kernel Path]]]
+
+Use -h for flag options
+`
+
+// Cmd executes the command line parsing for the add command
+func Cmd() {
+	if len(os.Args) < 3 {
```

Comment:
I started reviewing from below (because `main` was there) - the comments below about flag vs os.Args apply here and in the other files